x/emissions/types/params.go

```diff
@@ -27,8 +27,8 @@ func DefaultParams() Params {
 		PInferenceSynthesis:             alloraMath.NewDecFromInt64(2),             // free parameter used in the gradient function phi' for inference synthesis
 		PRewardSpread:                   alloraMath.NewDecFromInt64(1),             // fiducial value = 1; Exponent for W_i total reward allocated to reputers per timestep
 		AlphaRegret:                     alloraMath.MustNewDecFromString("0.1"),    // how much to weight the most recent log-loss differences in regret EMA update
-		MaxUnfulfilledWorkerRequests:    uint64(5),                                 // maximum number of outstanding nonces for worker requests per topic from the chain
-		MaxUnfulfilledReputerRequests:   uint64(5),                                 // maximum number of outstanding nonces for reputer requests per topic from the chain
+		MaxUnfulfilledWorkerRequests:    uint64(100),                               // maximum number of outstanding nonces for worker requests per topic from the chain
+		MaxUnfulfilledReputerRequests:   uint64(100),                               // maximum number of outstanding nonces for reputer requests per topic from the chain
```

Comment:
what's the rationale here? Would expect the old values...

Reply:
The rationale is that the unfulfilled worker nonces, with the advent of gt_lag, need to wait until they are requested so they can be fulfilled. 

So this number efffectively marks the max ratio that there could be `gt_lag / EpochLength`.
Bear in mind that this also means this needs to be set to keep all data from those blocks until they get fulfilled (which we need to).
So 100 is actually pretty conservative towards safety side imho, but the main point is that not only dev but also users  need to be aware as well.

Example: a 15 min freq for 24h prediction is already a ratio of 96.

Reply:
ok think i get. 100% needs to go in the docs